services/export/shared/requestAll.js

```diff
@@ -50,4 +49,60 @@ const requestAll = async (fn, params, limit) => {
 	return data;
 };
 
-module.exports = requestAll;
+const requestAllCustom = async (fn, method, params, limit) => {
+	const maxAmount = limit || 1e9;
+	const oneRequestLimit = params.limit || 100;
+	const firstRequest = await fn(method,
+		{
+			...params,
+			...{
+				limit: oneRequestLimit,
+				offset: 0,
+			},
+		});
+	const totalResponse = firstRequest;
+	if (!totalResponse.error) {
```

Comment:
Do we need to check if res is undefined here?
```suggestion
	if (totalResponse && !totalResponse.error) {
```